src/lib/radio/radio.ts

```diff
@@ -411,10 +417,19 @@ export class MdRadioButton implements OnInit {
       this.radioGroup._touch();
     }
   }
+
+  getHostElement() {
+    return this._elementRef.nativeElement;
+  }
+
+  isRippleEnabled() {
```

Comment:
We can bind directly to the disableRipple property. Just referred what button did for ripple. I can change that for button too.